templates/ecosystem/clients.html

```diff
@@ -1,5 +1,9 @@
 {% extends "page.html" %}
 {% import "macros/clients.html" as clients %}
+{% block head_extra %}
+<noscript><link rel="stylesheet" href="/no-js.css" /></noscript>
```

Comment:
Seems like you forgot to commit that file.

Reply:
I can see it in https://github.com/matrix-org/matrix.org/pull/1674/files#diff-8a85088eeb84225e3dc5b2229da18b417c810bc1ddb954f19e6acf201f5575a6, did I miss anything?

Reply:
Oh, I was searching for `no-js.css` but it's `no-js.scss`. Nevermind.

Reply:
FWIW I used `.scss` because I think Zola generates a minified css from the scss, but I didn't test if zola minifies a "raw" css file.

I wonder if we will need something to minify the js as well eventually.

Reply:
I think unless we write a lot more JS, gzip should be sufficient. Of course if Zola supports it out of the box it doesn't hurt to use that, but I don't think it does.

Reply:
From https://github.com/getzola/zola/issues/542#issuecomment-657169503 I'm not sure it does. But based on https://github.com/getzola/zola/issues/542#issuecomment-521409104, if it does it won't touch anything under `static`, so we need to move the js out of it. Maybe for a separate PR?